app/views/building_map/index.html.erb

```diff
@@ -43,21 +43,29 @@
 
 <datalist id="startOptions">
   <option><%= BuildingMapController::YOUR_LOCATION_MAGIC_STRING %></option>
+  <option><%= BuildingMapController::PIN_1_MAGIC_STRING %></option>
+  <option><%= BuildingMapController::PIN_2_MAGIC_STRING %></option>
   <%= Places::DESTINATIONS.each do |dest, coords| %>
     <option>
       <%= dest %>
     </option>
   <% end %>
 </datalist>
 <datalist id="destOptions">
+  <option><%= BuildingMapController::PIN_1_MAGIC_STRING %></option>
+  <option><%= BuildingMapController::PIN_2_MAGIC_STRING %></option>
   <%= Places::DESTINATIONS.each do |dest, coords| %>
     <option>
       <%= dest %>
     </option>
   <% end %>
 </datalist>
 
-<div class="container-fluid">
+<script>
+    var map;
+</script>
+
+<div class="container-fluid">git
```

Comment:
I'm not sure what the `git` string at the end of this line does 😅

Reply:
Neither do we :smile: